.github/workflows/release-notes.yaml

```
@@ -0,0 +1,76 @@
+
+      - name: Append release notes to changelog.md
+        run: |
+          sed -i "/<!-- append new version here -->/ a ## ${{ env.VERSION_WITH_PREFIX }}\n${{ env.NOTES }}\n" \
+            "${{ github.workspace }}/ebusd/CHANGELOG.md"
+          if [ $? -ne 0 ]; then
+            echo "Failed to update CHANGELOG.md"
+            exit 1
+          fi
+
+      - name: Update version in config.yaml
+        run: |
```

actionlint

shellcheck reported issue in this script: SC2181:style:3:6: Check exit code directly with e.g. 'if ! mycmd;', not indirectly with $?

(shellcheck)